pyfolio/pos.py

```diff
@@ -126,12 +133,18 @@ def extract_pos(positions, cash):
 
     positions = positions.copy()
     positions['values'] = positions.amount * positions.last_sale_price
+
     cash.name = 'cash'
 
     values = positions.reset_index().pivot_table(index='index',
                                                  columns='sid',
                                                  values='values')
 
+    if ZIPLINE:
+        for asset in values.columns:
+            if type(asset) in [Equity, Future]:
+                values[asset] = values[asset]*asset.price_multiplier
```

Comment:
Is this relevant also for Equities? Does `price_multiplier` always exist? Also needs a space around `*`.

Reply:
`price_multiplier` always exists. It is a [property](https://github.com/quantopian/zipline/blob/master/zipline/assets/_assets.pyx#L75) of the base class `Asset` from which `Equity` and `Future` are derived; it is initialized to 1.0. Yes, I think this is relevant in Equities. I can see you are testing [UK Equities](https://github.com/quantopian/zipline/blob/72993a3a4e6dfb58116663f56efd1ec1bd05bed9/zipline/algorithm.py#L2389). LSE stocks are quoted in GBp (pence); i.e., 1/100 of a GBP. I believe that the GBP notional equity position would be price\*shares\*multiplier, where price is in GBp, shares are as traded, and multiplier is 0.01.

Reply:
Thanks for explaining, makes sense.

Reply:
Then just fixing the pep8 style issue and we should be good.

Reply:
PEP8 style issue fixed.